graphsdk/src/main/java/com/microsoft/graph/model/Date.java

```diff
@@ -0,0 +1,90 @@
+package com.microsoft.graph.model;
+
+
+import java.util.Locale;
+
+/**
+ * A timezone-nonspecific date
+ */
+public class Date {
+
+    /**
+     * Array indices for date parsing
+     */
+    private static final int
```

Comment:
Just stick these as consts within the parse function since they are only used in one place.